tests/virtctl/credentials.go

```diff
@@ -91,22 +91,16 @@ var _ = Describe("[sig-compute][virtctl]credentials", func() {
 			))
 		})
 
-		keySecret := &corev1.Secret{
-			ObjectMeta: metav1.ObjectMeta{
-				Name: sshKeySecretName,
-				OwnerReferences: []metav1.OwnerReference{{
-					APIVersion: kubevirtv1.VirtualMachineGroupVersionKind.GroupVersion().String(),
-					Kind:       kubevirtv1.VirtualMachineGroupVersionKind.Kind,
-					Name:       vm.Name,
-					UID:        vm.UID,
-					Controller: pointer.Bool(true),
-				}},
-			},
-			Data: map[string][]byte{
-				"key-file.pub": []byte(testKey1),
-			},
-		}
-
+		keySecret := libsecret.New(sshKeySecretName, map[string][]byte{
+			"key-file.pub": []byte(testKey1),
+		})
+		keySecret.OwnerReferences = append(keySecret.OwnerReferences, metav1.OwnerReference{
```

Comment:
I do not see a reason to use `append` here.

Reply:
Thanks for the heads up, fixed.